js/viz/core/default.js

```diff
@@ -1457,6 +1457,94 @@ registerTheme({
                 alignment: RIGHT
             }
         }
+    },
+    funnel: {
+        sortData: true,
+        valueField: "value",
+        colorField: "color",
+        argumentField: "argument",
+        hoverEnabled: true,
+        selectionMode: "single",
+        item: {
+            border: {
+                width: 0,
+                color: WHITE
+            },
+            hoverStyle: {
+                hatching: {
+                    opacity: 0.75,
+                    step: 6,
+                    width: 2,
+                    direction: RIGHT
+                },
+                border: {
+                    width: 0,
+                    color: WHITE
+                }
+            },
+            selectionStyle: {
+                hatching: {
+                    opacity: 0.5,
+                    step: 6,
+                    width: 2,
+                    direction: RIGHT
+                },
+                border: {
+                    width: 0,
+                    color: WHITE
+                }
+            }
+        },
+        title: {
+            margin: 10
+        },
+        adaptiveLayout: {
+            width: 80,
+            height: 80,
+            keepLabels: true
+        },
+        legend: {
+            visible: false
+        },
+        _rtl: {
+            legend: {
+                itemTextPosition: LEFT
+            }
+        },
+        tooltip: {
+            customizeTooltip: function(info) {
+                return { text: info.item.data.argument + " " + info.valueText };
+            }
+        },
+        inverted: false,
+        algorithm: "dynamicSlope",
+        neckWidth: 0,
+        neckHeight: 0,
+        label: {
+            visible: true,
+            horizontalAlignment: RIGHT,
+            horizontalOffset: 0,
+            verticalOffset: 0,
+            showForZeroValues: true,
+            customizeText: function(info) {
+                return info.item.data.argument + " " + info.valueText;
```

Comment:
Why did you define default label text composition in through customizeText and in theme?